tests/istio.mk

```diff
@@ -59,12 +59,22 @@ EXTRA_E2E_ARGS += --ca_hub ${HUB}
 
 # A make target to generate all the YAML files
 generate_yaml:
-	./install/updateVersion.sh >/dev/null 2>&1
+	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
```

Comment:
cool this is a good improvement!

Reply:
There's already the "installgen" target, so another option is to delete the "generate_yaml" (but in any case the two can be consolitaed).  The piping to null should probably be controlled by the VERBOSE parameter.

Reply:
Found it - will replace both with a helm/install-gen based one, updateVersion is on its way out.